test/temperatureConverter.test.js

```diff
@@ -0,0 +1,10 @@
+import test from 'ava'
+import {temperatureConverter} from '../src'
+
+test('converts temperature', t => {
+  const number = 78
+  const expected = (number - 32) * 5/9
+  const actual = temperatureConverter(25.555555555555557)
```

Comment:
I think if you do: `const actual = temperatureConverter(number)` the test should pass here.

Reply:
thanks, test updated